cmd/api-resource-collector/scap.go

```diff
@@ -279,12 +254,34 @@ func (c *scapContentDataStream) SaveResources(to string) error {
 	return saveResources(to, c.found)
 }
 
-func saveResources(dir string, data map[string][]byte) error {
-	for f, d := range data {
-		err := ioutil.WriteFile(path.Join(dir, f), d, 0600)
+func saveResources(rootDir string, data map[string][]byte) error {
+	for apiPath, fileContents := range data {
+		saveDir, saveFile, err := getSaveDirectoryAndFileName(rootDir, apiPath)
+		if err != nil {
+			return err
+		}
+		err = os.MkdirAll(saveDir, os.ModePerm)
```

Comment:
why create the dirs with 777?

Reply:
Whoops, I left in that was my own laziness. Will follow-up